cli/pcluster/commands.py

```diff
@@ -133,12 +133,12 @@ def create(args):  # noqa: C901 FIXME!!!
         LOGGER.debug("StackId: %s", stack.get("StackId"))
 
         if not args.nowait:
-            utils.verify_stack_creation(cfn, stack_name)
+            utils.verify_stack_creation(stack_name, cfn)
```

Comment:
Rename `cfn` to `cfn_client` to stay consistent with `verify_stack_creation(stack_name, cfn_client=None)`

Reply:
Done